aspnetcore/blazor/security/webassembly/standalone-with-authentication-library.md

```diff
@@ -93,6 +93,8 @@ Authentication support for standalone apps is offered using OpenID Connect (OIDC
 
 ## Access token scopes
 
+The Blazor WebAssembly template automatically configures default scopes for `openid` and `profile`.
```

Comment:
Only for 3.x?

Reply:
Both 3.x and 5.x ...

https://github.com/dotnet/aspnetcore/blob/648c15dbe90fb8c113f7c6b4adeb40d9e10494f6/src/Components/WebAssembly/WebAssembly.Authentication/src/Options/OidcProviderOptions.cs#L33

https://github.com/dotnet/aspnetcore/blob/release/5.0-rc2/src/Components/WebAssembly/WebAssembly.Authentication/src/Options/OidcProviderOptions.cs#L33

... if I have the 5.0 branch correct. :smile:

Reply:
Ah, boo. This is the OIDC provider. I was thinking about a different codepath.